extensions/gitlab/src/components/project_search.tsx

```diff
@@ -22,11 +22,16 @@ import { ClearLocalCacheAction } from "./cache_actions";
 
 export function ProjectListItem(props: { project: Project }): JSX.Element {
   const project = props.project;
+  const accessories = [];
+  if (project.archived) {
+    accessories.push({ text: "Archived", icon: { source: Icon.ExclamationMark, tintColor: Color.Red } });
```

Comment:
@macbookandrew @andrewminion-luminfire Looks like it still has the text and is red, maybe forgot to push a commit?

Reply:
🤦🏻 yes, I forgot to push the commit. #2837 includes that change. Sorry!!

Reply:
already did…see #2837